public/userInfo.php

```diff
@@ -39,8 +44,7 @@
 $userPage = $userMassData['User'];
 $userMotto = $userMassData['Motto'];
 $userPageID = $userMassData['ID'];
-$setRequestList = getUserRequestList($userPage);
```

Comment:
This was the expensive call that was eliminated. This page only needs the number of used requests, not information about the games they're associated to. In particular, determining if the games were claimed was expensive.